README.md

```diff
@@ -2151,10 +2151,10 @@ no parameters.
   ```
 
 
-* <a name="camelcase-classes"></a>
-  Use `CamelCase` for classes and modules.  (Keep acronyms like HTTP, RFC, XML
+* <a name="pascalcase-classes"></a>
+  Use `PascalCase` for classes and modules.  (Keep acronyms like HTTP, RFC, XML
```

Comment:
Since both terms are used and understood separately by different people what do you think about using both names in the style guide?

``Use `UpperCamelCase` (PascalCase) for classes and modules.``